.goreleaser.yml

```diff
@@ -0,0 +1,46 @@
+project_name: sloop
+builds:
+  - id: sloop
+    main: ./pkg/sloop/main.go
+    binary: sloop
+    flags:
+      - -ldflags=-s
+      - -installsuffix=cgo
+    goos:
+      - linux
+      - darwin
+    goarch:
+      - amd64
+dockers:
+  - goos: linux
+    goarch: amd64
+    builds:
+      - sloop
+    binaries:
+      - sloop
+    extra_files:
+      - pkg/sloop/webfiles
+    image_templates:
+      - "salesforce/sloop:latest"
+      - "salesforce/sloop:{{ .Tag }}"
+    dockerfile: Dockerfile
+    build_flag_templates:
+      - "--pull"
+      - "--label=org.opencontainers.image.created={{.Date}}"
+      - "--label=org.opencontainers.image.name={{.ProjectName}}-captain"
+      - "--label=org.opencontainers.image.revision={{.FullCommit}}"
+      - "--label=org.opencontainers.image.version={{.Version}}"
```

Comment:
how these vars are populated? where do we get them from? e.g. {{.Version}} and {{.Tag}}

Reply:
@nurland goreleaser inherently has these variables -- https://goreleaser.com/customization/templates/